_posts/en/2019-10-21-schnorr-taproot-workshop.md

```diff
@@ -0,0 +1,190 @@
+  of the new schnorr and taproot technologies.
+
+Slides from all the presentations can be downloaded [here][slides].
+Additionally, Bryan Bishop has provided a [transcript][] of the New York
+session.
+
+## Introduction
+
+[![Introduction](/img/posts/taproot-workshop/introduction.png)](https://www.youtube.com/watch?v=1gRCVLgkyAE&list=PLPrDsP88ifOVTEJf_jQGunDUS05M9GdIC)
+{:.center}
+
+John Newbery gives a summary of why schnorr and taproot are useful technologies
+and explains why Bitcoin Optech created the workshop. He then outlines the
+workshop's aims.
+
+## Preparation and Basic Math
+
+This section shows how to set up the jupyter notebook, gives a refresher on
+basic elliptic curve math and introduces tagged hashes.
+
+#### 0.1 Test Notebook
+
+Before starting the workshops, users should follow the instructions in the
+repository [README][readme], clone the [workshop repository][] and run through
+the test notebook to ensure that their environment is set up correctly.
+
+#### 0.2 Elliptic Curve Math
+
+[![Introduction](/img/posts/taproot-workshop/elliptic-curve-math.png)](https://www.youtube.com/watch?v=oix8ov9iGgk&list=PLPrDsP88ifOVTEJf_jQGunDUS05M9GdIC&index=2)
+{:.center}
+
+Elichai Turkel provides a refresher on the basic elliptic curve math that will be
+required for this workshop.
+
+#### 0.3 Tagged Hashes
+
+_(No video)_ This chapter introduces _Tagged hashes_, which are used in both
+the bip-schnorr and bip-taproot proposals.
+
+## Schnorr signatures and MuSig
+
+This section explains the bip-schnorr proposal and explains how MuSig can be
+used to aggregate multiple public keys and partial signatures into a single
+pubkey and signature.
+
+#### 1.1 Schnorr Signatures
+
+[![Schnorr Signatures](/img/posts/taproot-workshop/schnorr.png)](https://www.youtube.com/watch?v=wybiVFdknhg&list=PLPrDsP88ifOVTEJf_jQGunDUS05M9GdIC&index=3)
+{:.center}
+
+Elichai explains the mathematics behind schnorr signatures and explains the
+bip-schnorr proposal.
+
+#### 1.2 MuSig
+
+[![MuSig](/img/posts/taproot-workshop/musig.png)](https://www.youtube.com/watch?v=5MbTptrXEC4&list=PLPrDsP88ifOVTEJf_jQGunDUS05M9GdIC&index=4)
+{:.center}
+
+Elichai describes the [MuSig algorithm][musig] (authored by Gregory Maxwell,
+Andrew Poelstra, Yannick Seurin and Pieter Wuille), and shows how it can be
+used to aggregate multiple public keys and partial signatures into a single
+pubkey/signature.
+
+## Taproot
+
+This section explains the bip-taproot and bip-tapscript proposals. It shows the
+format of a segwit v1 output and how such an output can be spent in a key path
+spend or a script path spend. It demonstrates how a tweaked public key can commit to
+one or more scripts, and how the segwit v1 output can be spent using one of those
+scripts.
+
+#### 2.0 Introduction to Taproot
+
+[![Introduction to Taproot](/img/posts/taproot-workshop/taproot-intro.png)](https://www.youtube.com/watch?v=KLNH0ttpdFg&list=PLPrDsP88ifOVTEJf_jQGunDUS05M9GdIC&index=5)
```

Comment:
This video title should have `(2.0)` appended to it

Reply:
the video doesn't actually correspond with notebook 2.0 (which was added after we ran the workshops). It's an introduction to all the 2.x topics.